.actrc

```diff
@@ -0,0 +1,3 @@
+-P ubuntu-latest=catthehacker/ubuntu:runner-latest
```

Comment:
why is this file being added?

Reply:
helps with local debugging of github actions workflow.